composer.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "vichan-devel/vichan",
+    "description": "vichan imageboard",
+    "type": "project",
+    "require": {
+        "ext-mbstring": ">=5.4",
+        "ext-gd": ">=5.4",
+        "ext-pdo": ">=5.4",
+        "twig/twig": "^1.44.2",
+        "lifo/ip": "^1.0",
+        "gettext/gettext": "^1.0",
+        "mrclay/minify": "^2.1.6",
+        "geoip/geoip": "^1.17"
+    },
+    "autoload": {
+        "classmap": ["inc/"],
+        "files": [
+            "inc/bootstrap.php",
+            "inc/display.php",
+            "inc/template.php",
+            "inc/database.php",
+            "inc/events.php",
+            "inc/api.php",
+            "inc/mod/auth.php",
+            "inc/lock.php",
+            "inc/queue.php",
+            "inc/polyfill.php",
+            "inc/functions.php"
+        ]
+    },
+    "license": "Tinyboard + vichan",
+    "authors": [
+        {
+            "name": "h00j",
+            "email": "admin@vichan.net"
```

Comment:
this is problematic per license

Reply:
ahh of course, what an oversight, thanks! resolved in https://github.com/vichan-devel/vichan/commit/3242e28b19dc9ff9c90383cd96167ba3c991c873

Reply:
no problem. i'm not sure if he's still alive, but when he was, @savetheinternet was extremely litigious. i remember well.